File: src/clients/dotnet/src/TigerBeetle/TransferFlags.cs
using System;

namespace TigerBeetle
{
	[Flags]
	public enum TransferFlags : ushort
	{
		None = 0,
		Linked = 1 << 0,
		Pending = 1 << 1,
		PostPendingTransfer = 1 << 2,
		VoidPendingTransfer = 1 << 3,
	}
}

File: src/clients/java/src/main/java/com/tigerbeetle/TransferFlags.java
package com.tigerbeetle;

public interface TransferFlags {
    short NONE = 0;
    short LINKED = (short) (1 << 0);
    short PENDING = (short) (1 << 1);
    short POST_PENDING_TRANSFER = (short) (1 << 2);
    short VOID_PENDING_TRANSFER = (short) (1 << 3);
}
